### PR TITLE
docs: cluster peering docs feedback

### PR DESCRIPTION
**Pull request by david-yu**
### Description

Small changes in cluster peering docs when going through Cluster Peering with @nathancoleman 

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/connect/cluster-peering/k8s.mdx [hunk 1]
```diff
@@ -423,10 +429,10 @@ To end a peering connection, delete both the `PeeringAcceptor` and `PeeringDiale
     1. Exec into the server pod for the first cluster. 
   
       ```shell-session
-      $ kubectl exec -it consul-server-0 -- /bin/sh
+      $ kubectl exec -it consul-server-0 --context $CLUSTER1_CONTEXT -- /bin/sh
       ```
   
-    1. Export an ACL token to access the `/health` HTP endpoint for services. The bootstrap token may be used if an ACL token is not already provisioned. 
+    1. If you've enabled ACL xport an ACL token to access the `/health` HTP endpoint for services. The bootstrap token may be used if an ACL token is not already provisioned.
```
Comment (nathancoleman) [inline website/content/docs/connect/cluster-peering/k8s.mdx:435]:
```suggestion
    1. If you've enabled ACLs, export an ACL token to access the `/health` HTP endpoint for services. The bootstrap token may be used if an ACL token is not already provisioned. 
```

**Review by nathancoleman — CHANGES_REQUESTED** [review]
One other thing that I ran into:

If I apply the annotation exactly from the code snippet [here](https://developer.hashicorp.com/consul/docs/connect/cluster-peering/k8s#recreate-or-reset-a-peering-connection), then I get this error every time

```shell
$ kubectl --context $CLUSTER1_CONTEXT apply --filename acceptor.yaml
error: unable to decode "acceptor.yaml": json: cannot unmarshal number into Go struct field ObjectMeta.metadata.annotations of type string
```

Instead, it needs to be `consul.hashicorp.com/peering-version: "1"` with quotes

**Comment by david-yu** [comment]
Thanks just update the docs, also followed up with @ndhanushkodi to see if this was a bug or area for improvement in the future.

**Comment by david-yu** [comment]
> One other thing that I ran into:
> 
> If I apply the annotation exactly from the code snippet [here](https://developer.hashicorp.com/consul/docs/connect/cluster-peering/k8s#recreate-or-reset-a-peering-connection), then I get this error every time
> 
> ```shell
> $ kubectl --context $CLUSTER1_CONTEXT apply --filename acceptor.yaml
> error: unable to decode "acceptor.yaml": json: cannot unmarshal number into Go struct field ObjectMeta.metadata.annotations of type string
> ```
> 
> Instead, it needs to be `consul.hashicorp.com/peering-version: "1"` with quotes

@nathancoleman I checked with Nitya and this is mainly a Kube limitation as in that values for annotations need to be strings, so not much we can do here other than docs.

**Review by nathancoleman — APPROVED** [review]

**Pull request merged**